dev/workflow/rocoto/gfs_tasks.py

```diff
@@ -2247,6 +2247,7 @@ def arch_tars(self):
 
             if self.options['do_ocean']:
                 tarball_types.extend(['ocean_6hravg', 'ocean_grib2', 'gfs_flux_1p00'])
+                tarball_types.append('gfsocean_analysis')
```

Comment:
This can be combined:
```suggestion
                tarball_types.extend(['ocean_6hravg', 'ocean_grib2', 'gfs_flux_1p00', 'gfsocean_analysis'])
```